examples/slack/README.md

```diff
@@ -0,0 +1,195 @@
+# Slack example
+
+This `repo.yaml` file contains a workflow specification to send Infracost cost estimates to Slack.
+
+Slack message blocks have a 3000 char limit so the Infracost CLI automatically truncates the middle of `slack-message` output format.
+
+## Table of Contents
+
+* [Running with GitHub](#running-with-github)
+* [Running with GitLab](#running-with-gitlab)
+* [Running with Azure Repos](#running-with-azure-repos)
+
+## Running with GitHub
+
+1. Update your setup to use the [infracost-atlantis](https://hub.docker.com/r/infracost/infracost-atlantis) Docker image.
+2. You'll need to pass the following custom env vars into the container:
+   ```sh
+   GITHUB_TOKEN=<your-github-token>
+   INFRACOST_API_KEY=<your-infracost-api-token>
+   SLACK_WEBHOOK_URL: <your-slack-webhook-url>
+   ```
+3. Add the following yaml spec to `repos.yaml` or `atlantis.yaml` config files:
+   ```yaml
+   repos:
+     - id: /.*/
+       workflow: terraform-infracost
+       post_workflow_hooks:
+         # Clean up any files left over from the run
+         - run: |
+             rm /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-infracost.json
+             rm /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-slack-message.json
+   workflows:
+     terraform-infracost:
+       plan:
+         steps:
+           - init
+           - plan
+           - show # this writes the plan JSON to $SHOWFILE
+           # Run Infracost breakdown and save to a tempfile, namespaced by this project, PR, workspace and dir
+           - run: |
+               infracost breakdown --path=$SHOWFILE \
+                                   --format=json \
+                                   --log-level=info \
+                                   --out-file=/tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-infracost.json
+           # Use Infracost comment to create a comment containing the results for this project
+           - run: |
+               infracost comment github --repo $BASE_REPO_OWNER/$BASE_REPO_NAME \
+                                        --pull-request $PULL_NUM \
+                                        --path /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-infracost.json \
+                                        --behavior new \
```

Comment:
As agreed, the default behavior is `new`. I'll update it in other examples when add cases for Azure Repos.